src/ProductTour/ProductTour.test.jsx

```diff
@@ -132,17 +127,58 @@ describe('<ProductTour />', () => {
         expect(screen.getByRole('dialog', { name: 'Checkpoint 1' })).toBeInTheDocument();
 
         // Click the dismiss button
-        const dismissButton = screen.getByRole('button', { name: 'Dismiss' });
-        expect(dismissButton).toBeInTheDocument();
+        const closeButton = screen.getByRole('button', { name: 'Close tour' });
+        expect(closeButton).toBeInTheDocument();
 
         await act(async () => {
-          await userEvent.click(dismissButton);
+          await userEvent.click(closeButton);
         });
 
         // Verify no Checkpoints have rendered
         expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
       });
 
+      it('onClick of close icon button disables tour', async () => {
+        const { rerender } = render(<ProductTourWrapper tours={[tourData]} />);
+        // Advance the tour, close icon only appears after first step
+        await act(async () => {
+          await userEvent.click(screen.getByRole('button', { name: 'Next' }));
+        });
+        rerender(<ProductTourWrapper tours={[tourData]} />);
+
+        const closeIcon = screen.getByRole('button', { name: 'Close tour' });
```

Comment:
If we move the translation from the [CheckpointHeader.jsx](https://github.com/kiram15/paragon/blob/kiram15/ENT-10146-1/src/ProductTour/CheckpointHeader.jsx#L17) component to a translation file (messages.js), we will be able to use this translation in our tests.